hs_access_control/migrations/0021_auto_20170418_2042.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
@zhiyuli Yeah for proxy models we can't have any database fields. But I am not sure why Django created this new migration file in hs_access_control app and the contents of this migration file also doesn't make much sense. Need some thinking...